.goreleaser.yml

```diff
@@ -18,8 +18,6 @@ builds:
 dockers:
   - goos: linux
     goarch: amd64
-    binaries:
```

Comment:
[the docs](https://goreleaser.com/deprecations/#dockerbinaries) say to replace `binaries` with `ids`, not sure if removal will not cause an issue

Reply:
I tried to replace it first, as the docs say, but it caused another issue. Removal worked for me both locally and here.